contrib/systemd/cilium-health

```diff
@@ -0,0 +1,24 @@
+### Configuration options for cilium-health
```

Comment:
@nirmoy @darix Is this format correct? PTAL

Reply:
LGTM